cl/settings/django.py

```diff
@@ -56,9 +56,9 @@
 ####################
 CACHES = {
     "default": {
-        "BACKEND": "redis_cache.RedisCache",
-        "LOCATION": f"{REDIS_HOST}:{REDIS_PORT}",
-        "OPTIONS": {"DB": REDIS_DATABASES["CACHE"], "MAX_ENTRIES": 1e5},
+        "BACKEND": "django.core.cache.backends.redis.RedisCache",
+        "LOCATION": f"redis://{REDIS_HOST}:{REDIS_PORT}",
+        "OPTIONS": {"db": REDIS_DATABASES["CACHE"]},
```

Comment:
The options array format is different, that's what was causing the test failure.

Reply:
Best I can tell `MAX_ENTRIES` doesn't really have an equivalent replacement option in the `django.core.cache.backends.redis.RedisCache` backend.

Reply:
Yeah, you're right. I checked the code. I checked django-redis-cache too, and it didn't support it either. Looks like we've been living without.